### ISSUE TITLE
Stuck installing TensorRT

### ISSUE THREAD

**Issue by elekktronaut** [olegchomp/TouchDiffusion#4]
Installing worked well until here:

Installing TensorRT requirements...
<module 'tensorrt' from 'D:\\TOUCHDESIGNER\\External\\olegchomp\\TouchDiffusion-main\\.venv\\Lib\\site-packages\\tensorrt\\__init__.py'>

Nothing seems to happen. I've tried restarting but that doesn't help :/

**Comment by virusvjvisuals** [comment]
same for me, 
 :<module 'tensorrt' from 'D:\TOUCHDESIGNER\External\olegchomp\TouchDiffusion-main\.venv\Lib\site-packages\tensorrt\init.py'> , but having a look to the webui,  it says that it s all done and that you need to restart webui.bat to apply changes... 
 I tried 3 times relaunching having the same result, even now doing a complete reinstall from scratch.. 
 Is there a way to check about the completion of the process ?

**Comment by elekktronaut** [comment]
It actually still works :) Running with 17 FPS on my 2070 laptop!!

**Comment by olegchomp** [comment]
> It actually still works :) Running with 17 FPS on my 2070 laptop!! 
> 
> Edit: Okay more like 7 FPS

In webui it show pure speed without preprocessing images. It should be slower, for few FPS, but not 10 I think. Might be throttling?

**Comment by olegchomp** [comment]
> same for me, 
> 
>  :<module 'tensorrt' from 'D:\TOUCHDESIGNER\External\olegchomp\TouchDiffusion-main\.venv\Lib\site-packages\tensorrt\init.py'> , but having a look to the webui,  it says that it s all done and that you need to restart webui.bat to apply changes... 
> 
>  I tried 3 times relaunching having the same result, even now doing a complete reinstall from scratch.. 
> 
>  Is there a way to check about the completion of the process ?
> 
> (just a detail : I did a model conversion and got a "TensorRT acceleration enabled" in the end of the model conversion, so hope this works !! ^^)

Yes, if engine created, mostly means that everything should work in TD

**Comment by olegchomp** [comment]
> It actually still works :) Running with 17 FPS on my 2070 laptop!! 
> 
> Edit: Okay more like 7 FPS

Updated benchmarks with 3060 12G and 16 FPS. I think 2070 should be close this values also, so might be something with hardware or throttling.

**Comment by superlotek** [comment]
I keep getting stuck on installing TensorRT as well.
Getting this error when I try to Update Dependencies:
Error installing packages: torch==2.1.0, onnx==1.15.0, onnxruntime==1.16.3, Failed to install TensorRT
